core/lib/i18n/i18n.js

```diff
@@ -189,8 +189,14 @@ function createIcuMessageFn(filename, fileStrings) {
     const keyname = Object.keys(mergedStrings).find(key => mergedStrings[key] === message);
     if (!keyname) throw new Error(`Could not locate: ${message}`);
 
-    const filenameToLookup = keyname in fileStrings ? filename : getModulePath(import.meta);
-    const unixStyleFilename = path.relative(LH_ROOT, filenameToLookup).replace(/\\/g, '/');
+    let filenameToLookup = keyname in fileStrings ? filename : getModulePath(import.meta);
```

Comment:
btw, any idea why this check/fallback to `getModulePath` is needed?

couldn't find any reasoning looking at the initial commit https://github.com/GoogleChrome/lighthouse/pull/5644/files#diff-1ca0a127c39d556edaeb5e53e6a12b74eaf76095850329302e48b24af41225d5R76

and i18n unit tests do not fail without this code.

Reply:
Ah, it allows usages of `str_` to reference i18n strings not defined in the "local context".

<img width="643" alt="image" src="https://user-images.githubusercontent.com/4071474/186268821-142cbb7e-3edb-4d74-9528-4a37e476626c.png">

Reply:
Attempted a more direct/readable way to do the same.